target/linux/rockchip/patches-6.1/100-rockchip-use-system-LED-for-OpenWrt.patch

```diff
@@ -27,25 +35,29 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
  	};
  
  	chosen {
-@@ -49,18 +54,18 @@
+@@ -48,19 +54,22 @@
+ 		pinctrl-names = "default";
  
  		lan_led: led-0 {
++			color = <LED_COLOR_ID_GREEN>;
++			function = LED_FUNCTION_LAN;
```

Comment:
Why doesn't the upstream dts file contain the color and function property?

Reply:
When it was merged upstream it still used label binding.